src/contour/Config.h

```diff
@@ -180,11 +180,10 @@ struct MouseConfig
 struct IndicatorConfig
 {
     std::string left { " {InputMode:Bold,Color=#FFFF00}"
-                       "{Tabs:ActiveColor=#FFFF00,Left= │ }"
                        "{SearchPrompt:Left= │ }"
                        "{TraceMode:Bold,Color=#FFFF00,Left= │ }"
                        "{ProtectedMode:Bold,Left= │ }" };
-    std::string middle { "{Title:Left= « ,Right= » }" };
+    std::string middle { "{Tabs: ActiveColor=#FFFF00}" };
```

Comment:
Let's figure out what this space is used for here :)

Reply:
removed it